src/main/java/com/algorithmia/algorithmHandler/Response.java

```diff
@@ -0,0 +1,46 @@
+package com.algorithmia.algorithmHandler;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import org.apache.commons.codec.binary.Base64;
+
+class Response<T> {
+    private MetaData metaData;
+    private T result;
+    public class MetaData{
+        private String content_type;
+        MetaData(String contentType){this.content_type = contentType;}
+    }
+    Response(Object rawData){
```

Comment:
Let's use a more tight type here than `Object`. It seems like we know that at the very least, the data is represented in a byte array; we could use `byte[]`.

Reply:
This might be a bit tricky, I'll spend some time thinking about how to refactor this into something a bit cleaner.